recipes/nbval/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "nbval" %}
+{% set version = "0.5" %}
+{% set sha256 = "bd245c0fb3ade4aff63948d31de5cb5f92129c7387e597d1229f687e5350eba7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
```

Comment:
There is a discussion about this topic as well

https://github.com/conda-forge/staged-recipes/issues/528